### ISSUE TITLE
Build multiarch docker images

### ISSUE THREAD

**Issue by onedr0p** [sstarcher/helm-exporter#34]
This would be nice for running helm-exporter on ARM, ARM64, or other architectures. For a head-start, check out my exporter and the Github actions here which implement this.

https://github.com/onedr0p/radarr-exporter

As you can see below my builds are multi-arch

![image](https://user-images.githubusercontent.com/213795/78507507-5971b500-774e-11ea-983d-21a7ba30c3d7.png)

**Comment by sstarcher** [comment]
Got ya, are you trying to run this on ARM?

**Comment by onedr0p** [comment]
Not at the moment, but it shouldn't be hard to add.

**Comment by rlex** [comment]
stumbled upon same issue while trying to run on arm64. Will be nice to have multiarch builds indeed.

**Comment by rlex** [comment]
here is adopted multiarch dockerfile:
```
FROM --platform=$BUILDPLATFORM golang:1.17 as builder

WORKDIR /go/src/github.com/sstarcher/helm-exporter
COPY . /go/src/github.com/sstarcher/helm-exporter
ARG TARGETOS TARGETARCH
RUN CGO_ENABLED=0 GOOS=$TARGETOS GOARCH=$TARGETARCH go build -o /go/bin/helm-exporter /go/src/github.com/sstarcher/helm-exporter/main.go

FROM alpine:3
RUN apk --update add ca-certificates
RUN addgroup -S helm-exporter && adduser -S -G helm-exporter helm-exporter
USER helm-exporter
COPY --from=builder /go/bin/helm-exporter /usr/local/bin/helm-exporter

ENTRYPOINT ["helm-exporter"]
```

Works for amd64 and arm64 with 
```
docker buildx build --platform linux/amd64,linux/arm64,linux/arm/v7 -t username/helm-exporter:1.1.1 --push .
```

**Comment by sstarcher** [comment]
If someone wants to write the github actions to automate that build process I would be happy to accept the PR.

**Comment by rlex** [comment]
i'll try it a bit later, looks like a nice exercise

**Comment by rlex** [comment]
@sstarcher looks like you was faster than me.
But currently, arm64 image contains amd64 build

The reason is here: https://github.com/sstarcher/helm-exporter/blob/master/Dockerfile#L6

GOARCH still points to hardcoded amd64 
replacing it with $TARGETARCH should do the trick

**Comment by sstarcher** [comment]
Want to toss me a pr?

**Comment by rlex** [comment]
sure thing, done

**Comment by sstarcher** [comment]
This should be resolved in 1.2.2
